htdocs/user/card.php

```diff
@@ -2043,9 +2043,9 @@
 			// Other form for user password
 			$parameters = array('valuetoshow' => $valuetoshow, 'caneditpasswordandsee' => $permissiontoeditpasswordandsee, 'caneditpasswordandsend' => $permissiontoeditpasswordandsend);
 			$reshook = $hookmanager->executeHooks('printUserPasswordField', $parameters, $object, $action); // Note that $action and $object may have been modified by hook
-			if ($reshook > 0) {
+			if ($reshook === 1) {
```

Comment:
The dev rules for hook are
Return 0 means the hook add action
Return > 0 means replace action
So we should have > 0 here

Reply:
@eldy ok it's removed